packages/forms/src/patterns/repeater/prompt.ts

```diff
@@ -0,0 +1,28 @@
+import { type RepeaterPattern } from './index.js';
+import {
+  type CreatePrompt,
+  type RepeaterProps,
+  createPromptForPattern,
+  getPattern,
+} from '../../index.js';
+
+export const createPrompt: CreatePrompt<RepeaterPattern> = (
+  config,
+  session,
+  pattern,
+  options
+) => {
+  const children = pattern.data.patterns.map((patternId: string) => {
+    const childPattern = getPattern(session.form, patternId);
+    return createPromptForPattern(config, session, childPattern, options);
+  });
```

Comment:
One more question. As I understand, the value of each child pattern is stored as a `repeater` attribute on the child pattern's session data. Is that right?

If so, I'm wondering if altering the structure of the child pattern's user data might be confusing down the road, and if it would be more clear to store user session data on the repeater pattern's session object. So essentially, the repeater's value might be something like:

```json
[
  {"patternIdA": "first row a", "patternIdB": "first row b"},
  {"patternIdA": "second row a", "patternIdB": "second row b"},
  {"patternIdA": "third row a", "patternIdB": "third row b"},
]
```